features/facebookImport/src/importer/importer-util.js

```diff
@@ -21,6 +24,14 @@ async function readJSONFile(dataFileName, zipFile, zipId = null) {
     if (!dataZipEntry) {
         throw new MissingFileImportException(dataFileName);
     }
+    const firstStat = await zipFile.stat(dataZipEntry);
+
+    const fileSize =
+        "size" in firstStat ? parseInt(firstStat.size) : firstStat.getSize();
```

Comment:
This is weird. So firstStat can have two different types? Not in scope, but shouldn't this be handled at the polyPod level?

Reply:
That is pretty odd.... So in `RemoteClientPod`,  there is a property called `size` - so this will work. But how I see it, it should have the exact same effect as just calling `getSize` in the first place. Quick comment on why this is needed (or just make it a TODO comment :P) would do for now IMHO.

Reply:
Added a TODO comment - that makes this mergeable IMO.

Reply:
Added a TODO for now